public/assets/scripts/.eslintrc.js

```diff
@@ -1,24 +1,27 @@
-
 // get polyfill settings from top level config
 // @ts-ignore
 const { settings } = require('../../../.eslintrc.json');
 
 // Adding non-polyfilable Symbol-related functions as they are most probably
 // behind the flag
 
-settings.polyfills.push('Symbol.toStringTag', 'Symbol.for', 'Object.getOwnPropertySymbols', 'Object.getOwnPropertyDescriptors')
+settings.polyfills.push(
+  'Symbol.toStringTag',
```

Comment:
Do we actually use `Symbol` anywhere? 🤔

Reply:
Sure. Your favorite Array.from is based on Symbol.Iterable 😉

Reply:
But those here are due to webpack